index.html

```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<!-- Nutrition Label -->

```

Comment:
I am going to start a code review

Reply:
Ok, I see the comments you added!  Can you merge?